zerolog/options_test.go

```diff
@@ -0,0 +1,173 @@
+package hertz_contrib_zerolog
+
+import (
+	"bytes"
+	"github.com/cloudwego/hertz/pkg/common/hlog"
+	"github.com/cloudwego/hertz/pkg/common/json"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+)
+
```

Comment:
I think this might be better
```go
import (
	"bytes"
	"path/filepath"
	"strings"
	"testing"
	"time"

	"github.com/cloudwego/hertz/pkg/common/hlog"
	"github.com/cloudwego/hertz/pkg/common/json"
	"github.com/rs/zerolog"
	"github.com/stretchr/testify/assert"
)
```

Reply:
Agreed. I have made changes to place url imports at the bottom.

Reply:
you can click https://github.com/hertz-contrib/logger/invitations